packages/mux-player/src/themes/gerwig/gerwig.html

```diff
@@ -66,15 +66,27 @@
      * 0.433s is the transition duration for VTT Regions.
      * Borrowed here, so the captions don't move too fast.
      */
-    media-controller ::slotted([slot='media']) {
```

Comment:
NOTE: Although this change isn't technically required, custom CSS properties (aka "CSS variables") inherit, so removing the narrower selection since it's both unnecessary and adds specificity that is harder to override.